pkg/server/wireexts_oss.go

```diff
@@ -8,8 +8,6 @@ import (
 
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/plugins"
```

Comment:
This one still here due to below

```go
wire.Bind(new(plugins.RoleRegistry), new(*acimpl.Service)),
```

I think it would make sense to have the implementation of plugins.RoleRegistry in the pluginsintegration package rather than in accesscontrol since it's an integration point between the two. Might be something to pick up later or what do you think?